src/mem/mem.c

```diff
@@ -68,45 +68,80 @@ static inline void mem_unlock(void)
 }
 
 /** Update statistics for mem_zalloc() */
-#define STAT_ALLOC(m, size) \
+#define STAT_ALLOC(_m, _size) \
```

Comment:
can you please make the renaming of _m a separate PR ?

Reply:
The renaming is necessary for this PR. `size` argument aliases the `size` member of the `mem` struct, without renaming the modified code does not compile.